tools/lkl/lib/virtio_net_dpdk.c

```diff
@@ -33,9 +33,13 @@ static char *ealargs[4] = {
 /* XXX: disable cache due to no thread-safe on mempool cache. */
 #define MEMPOOL_CACHE_SZ        0
 /* for TSO pkt */
-#define MAX_PACKET_SZ           (65535 \
+/*#define MAX_PACKET_SZ           (65535 \*/
+#define MAX_PACKET_SZ           (16383 \
 	- (sizeof(struct rte_mbuf) + RTE_PKTMBUF_HEADROOM))
+#if 0
 #define MBUF_NUM                (512*2) /* vmxnet3 requires 1024 */
+#endif //0
+#define MBUF_NUM                4096
```

Comment:
thanks, so you're using Intel i350 NIC (igb driver).
The previous value (512*2) works for my testing (ixgbe and vmxnet3 drivers).

I am not aware of a transparent solution of this variable for the moment.  In such case, comment lines stating is would be nice.

I will also look at existing examples (of DPDK) how they handle this.